spec/test_app/app/controllers/basechurch/v1/announcements_controller_spec.rb

```diff
@@ -19,14 +26,18 @@
 
   shared_examples_for 'a response containing an announcement' do
     it 'returns an announcement' do
-      body = JSON.parse(response.body)
-
-      expect(body['announcements']['id']).to eq(announcement.id.to_s)
-      expect(body['announcements']['description']).to eq(announcement.description)
-      expect(body['announcements']['position']).to eq(announcement.position)
-      expect(body['announcements']['url']).to eq(announcement.url)
-      expect(body['announcements']['links']['bulletin']).to eq(announcement.bulletin_id.to_s)
-      expect(body['announcements']['links']['post']).to eq(announcement.post_id.to_s)
+      data = JSON.parse(response.body)["data"]
+      attributes = data["attributes"]
+
+      expect(data["id"]).to eq announcement.id.to_s
+      expect(data["type"]).to eq "announcements"
+      expect(attributes['description']).to eq announcement.description
+      expect(attributes['position']).to eq announcement.position
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.